lib/uber/inheritable_attr.rb

```diff
@@ -1,6 +1,7 @@
 module Uber
   module InheritableAttr
-    def inheritable_attr(name)
+
+    def inheritable_attr(name, options={})
       instance_eval %Q{
```

Comment:
Right. Guess I need some more conference exp to code efficiently after some beers ;) Anyway, I've rewritten that and pushed here.